### ISSUE TITLE
Items Does not update dynamically when adding

### ISSUE THREAD

**Issue by naren0021** [sag1v/react-elastic-carousel#131]
I am trying to create a slider dynamically, I am getting the updated array in the react component but the slider doesn't re-render and show the items. I want to show itemsToShow = 5 
[Check Link Video](https://www.awesomescreenshot.com/video/2009160?key=c33bdf02194a0c84826243a920f8fff7)

sample Code 

```
import React, { Component } from 'react';
import Carousel, { consts } from 'react-elastic-carousel';

const breakPoints = [
    { width: 1, itemsToShow: 1 },
    { width: 550, itemsToShow: 2, itemsToScroll: 2 },
    { width: 850, itemsToShow: 3, itemsToScroll: 3 },
    { width: 1150, itemsToShow: 4, itemsToScroll: 4 },
    { width: 1450, itemsToShow: 5, itemsToScroll: 5 },
    { width: 1750, itemsToShow: 6, itemsToScroll: 6 }
];
var settings = {
    breakPoints: breakPoints,
    pagination: false,
    showArrows: true,
    infinite: false,
    transitionMs: 500,
    itemsToShow: 5,
    itemsToScroll: 1,
    enableSwipe: true
};

class SpeakerList extends Component {
    constructor(props) {
        super(props)
        this.state = {
            initialSpeakerList: [],
            showSpeakerViewModal: false,
            speakerItem: '',
            currentIndex: 0,
            speakerName: ''
        }
    }

    componentDidMount() {
        this.setState({ initialSpeakerList: JSON.parse(JSON.stringify(this.props.speakerList)), speakerName: this.props.speakerName })
    }

    componentDidUpdate(prevProps) {
        if (JSON.stringify(prevProps.speakerList) !== JSON.stringify(this.props.speakerList)) {
            this.setState({ initialSpeakerList: JSON.parse(JSON.stringify(this.props.speakerList)) }, () =>
                console.log("initialSpeakerList", this.state.initialSpeakerList)
            )
        }
        if (prevProps.speakerName !== this.props.speakerName) {
            this.setState({ speakerName: this.props.speakerName })
        }
    }

    setSlide = (currentIndex, slidesLength) => {
        const sliderIndex = ((currentIndex + 1) % slidesLength);
        this.setState({ currentIndex: sliderIndex })
    }

    render() {
        const { initialSpeakerList, currentIndex, speakerName } = this.state;
        return (
            <div className="speaker-carousel-list slider">
                <div className="speaker-list-header">
                    <h3 className="title">{speakerName}</h3>
                    {initialSpeakerList && initialSpeakerList.length && initialSpeakerList.length > 5 ?
                        <button className="btn-next-slide"
                            onClick={() => this.setSlide(currentIndex, initialSpeakerList.length)}>
                            <i class="icomoon icon-right-arrow-new"></i>
                        </button>
                        : null}
                </div>
                <Carousel renderArrow={this.myArrow} {...settings}>
                    {
                        initialSpeakerList.map((item =>
                            <div className="speaker-item">
                                <figure>
                                    <img src={item.event_speakerImage} />
                                    <span className="hover"><i class="icomoon icon-Search f32 v-middle"></i></span>
                                </figure>
                                <div className="detail">
                                    <p className="name">{item.event_speakerName}</p>
                                    <p className="position">{item.event_speakerRole}</p>
                                </div>
                            </div>
                        ))
                    }
                </Carousel>
            </div>
        )
    }
}

export default SpeakerList;
```

**Comment by sag1v** [comment]
Hi @naren0021 Thanks. Can you try install a beta version and try it?

Try installing `react-elastic-carousel@next`

**Comment by naren0021** [comment]
after using the beta version I am still facing the same issue

**Comment by sag1v** [comment]
Strange, it seems to work fine in [this example](https://codesandbox.io/s/react-elastic-carousel-forked-vv9r1?file=/src/index.js).

Make sure you got the version `0.10.0-beta.1`

**Comment by naren0021** [comment]
Thanks Man!! Really appreciated it works using the beta version.

**Comment by sag1v** [comment]
available as of version v `0.10.0`

**Comment by naren0021** [comment]
Thanks brother
